packages/client/src/components/topicList/TopicList.tsx

```diff
@@ -0,0 +1,38 @@
+import cn from 'classnames'
+import { topics, TTopic } from '@/mock/index'
+import styles from './TopicList.module.scss'
+import { Topic } from '../topic/Topic'
+import { Button } from '../ui/Button/Button'
+
+type TTopicListProps = {
+  selectedTopic: TTopic | null
+  setSelectedTopic: React.Dispatch<React.SetStateAction<TTopic | null>>
+  openModal: () => void
```

Comment:
[Предложение]
Мне нравится записывать тип функции вот так:
```suggestion
  openModal(): void
```
Так функции в ide часто подсвечиваются иначе, их проще отделить глазами от переменных